### ISSUE TITLE
Looks like a solid project

### ISSUE THREAD

**Issue by neurok8050** [brainhack-school2022/dimitrijevic_project#3]
I was wandering if the machine learning you wanted to do was part of the PCN toolkit or machine learning tools outside the PCN toolkit. Because, maybe you could also compare the classification result from the PCN toolkit with the classification result from another machine learning classification outside the PCN toolkit.

Pretty interesting project!

**Comment by Andjelaaaa** [comment]
Yes, I was mostly referring to the one included in the PCN toolkit, but yeah that's a great idea to try and compare it to other machine learning based regression models!
